src/os/ObjectStore.cc

```diff
@@ -146,8 +146,13 @@ ostream& operator<<(ostream& out, const ObjectStore::Sequencer& s)
   return out << "osr(" << s.get_name() << " " << &s << ")";
 }
 
+ostream& operator<<(ostream& out, const ObjectStore::Transaction& tx) {
+
+  return out << "Transaction address = " << &tx;
```

Comment:
Yes, I kept it like this since the old code was also printing transaction address only..